src/sentry/seer/anomaly_detection/get_historical_anomalies.py

```diff
@@ -32,29 +36,110 @@
 )
 
 
+def handle_seer_error_responses(response, config, context, log_params):
+    def log_statement(log_level, text, extra_data=None):
+        log_data = {**log_params}
+        if extra_data:
+            log_data.update(**extra_data)
+        if log_level == "error":
+            logger.error(text, extra=log_data)
+        elif log_level == "warning":
+            logger.warning(text, extra=log_data)
```

Comment:
hmm, in retrospect this is probably not adding a lot since we can't do the JS thing of just treating everything like an object 🤣

up to you for changing this back to just using logger stuff instead of the `log_statement`

---

that said, 💯 on the rest of this refactor. looks great.